lib/health-data-standards/models/entry.rb

```diff
@@ -24,12 +23,12 @@ class Entry
   field :negationReason, as: :negation_reason, type: Hash
   field :oid, type: String
   field :reason, type: Hash
-  
-  attr_protected :version
-  attr_protected :_id
-  attr_protected :created_at
-  attr_protected :updated_at
-  
+
+  attr_accessor :version
+  attr_accessor :_id
+  attr_accessor :created_at
+  attr_accessor :updated_at
```

Comment:
I'll chalk that up to battle scars from working through all the various generations of Rails.